geolocation/src/web.ts

```diff
@@ -0,0 +1,77 @@
+import { WebPlugin } from '@capacitor/core';
+
+import type {
+  GeolocationPlugin,
+  GeolocationOptions,
+  GeolocationPosition,
+  GeolocationWatchCallback,
+  GeolocationPermissionStatus,
+} from './definitions';
+
+export class GeolocationWeb extends WebPlugin implements GeolocationPlugin {
+  async getCurrentPosition(
+    options?: GeolocationOptions,
+  ): Promise<GeolocationPosition> {
+    return new Promise((resolve, reject) => {
+      navigator.geolocation.getCurrentPosition(
+        pos => {
+          resolve(pos);
+        },
+        err => {
+          reject(err);
+        },
+        {
+          enableHighAccuracy: true,
```

Comment:
```suggestion
          enableHighAccuracy: false,
```
the documented value for `enableHighAccuracy` is false